python/cudf/_gdf.py

```diff
@@ -594,3 +594,31 @@ def quantile(column, quant, method, exact):
                                        gdf_context)
         res.append(px[0])
     return res
+
+
+def digitize(column, bins, result, right=False):
```

Comment:
Could you do this in Cython instead of via CFFI? We want to phase CFFI out moving forward.

Reply:
Done!